src-docs/src/views/guidelines/getting_started/getting_started.js

```diff
@@ -0,0 +1,255 @@
+            <p>If you provide both, it will duplicate the imported styles.</p>
+            <h4>Touch the least amount of variables possible.</h4>
+            <p>
+              By nature EUI is very rigid. You shouldn&apos;t need much to make
+              drastic changes to color. Most themes are less then a dozen
+              variable overwrites in total.
+            </p>
+            <h4>
+              Do not overwrite individual component variables or{' '}
+              <EuiCode>.eui</EuiCode> class names.
+            </h4>
+            <p>
+              Although this may be possible, components are much more prone to
+              change and you&apos;ll risk breaking your theme. All EUI
+              components accept custom a <EuiCode>className</EuiCode> which you
+              can use to append your custom styles.
+            </p>
+          </EuiText>
+        </>
+      ),
+    },
+    {
+      title: 'Fonts',
+      wrapText: false,
+      text: (
+        <>
+          <EuiText grow={false}>
+            <p>
+              By default, EUI ships with a font stack that includes some
+              outside, open source fonts. If your system is internet available
+              you can include these by adding the following imports to your
+              SCSS/CSS files, otherwise you&#39;ll need to bundle the physical
+              fonts in your build. EUI will drop to System Fonts (which you may
+              prefer) in their absence.
+            </p>
+          </EuiText>
+          <EuiSpacer />
+
+          <EuiCodeBlock language="scss" isCopyable fontSize="m">
+            {
+              "@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&family=Roboto+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap');"
+            }
+          </EuiCodeBlock>
+          <EuiSpacer />
+          <EuiText grow={false}>
+            <p>
+              Or grab all weights, including italics, between 400 and 700 as a
+              variable font.
+            </p>
+          </EuiText>
+          <EuiSpacer />
+          <EuiCodeBlock language="scss" isCopyable fontSize="m">
+            {
+              "@import url('https://fonts.googleapis.com/css2?family=Inter:slnt,wght@-10,300..700;0,300..700&family=Roboto+Mono:ital,wght@0,400..700;1,400..700&display=swap');"
+            }
+          </EuiCodeBlock>
+
+          <EuiSpacer />
+
+          <EuiText grow={false}>
+            <h3 id="embedding-with-font-face-">
+              Embedding with <EuiCode>@font-face</EuiCode>
+            </h3>
+            <p>
+              If your application doesn&#39;t allow grabbing the font assets
+              from a CDN, you&#39;ll need to download and locally provide the
+              font files. You should download the files directly from their
+              source site <a href="https://rsms.me/inter/">rsms.me/inter/</a>.
+              Then follow the instructions in the provided CSS file for
```

Comment:
Minor grammar suggestion
```suggestion
              font files. You can download the files directly from their
              source site <a href="https://rsms.me/inter/">rsms.me/inter/</a>,
              then follow the instructions in the provided CSS file for
```